report-results/docs/index.md

```diff
@@ -525,11 +521,44 @@ The API returns an `HTTP 200 OK` response if the report is unavailable. The `err
 
 | Possible Cause | Solution |
```

Comment:
Maybe use [this tool](https://www.tablesgenerator.com/markdown_tables#:~:text=Markdown%20tables%20support,syntax%20for%20creating%20simple%20tables.) to make the table look better?

Reply:
sure go ahead

Reply:
Okay, go ahead and merge it and I'll update the table after